subgraphs/rocket-pool/networks.json

```diff
@@ -0,0 +1,7 @@
+{
+  "mainnet": {
+    "rocketpool": {
+      "address": "0x4D05E3d48a938db4b7a9A59A802D5b45011BDe58"
+    }
+  }
+}
```

Comment:
Actually not super familiar with deployment frameworks / what this is used for, so I'd defer to you for guidance here. Whatever makes the most sense. @jaimehgb

Reply:
The way it is done in most other subgraphs is by having the yaml being a template, and then filling the contract addresses and start blocks from a config.json file that is different for each network.

Take a look at badgerdao's subgraph for example (only has 1 deployment): 
- `/config/deployments/badgerdao-ethereum/configurations.json` -> values used to fill the template
- `/config/deployments/templates/badgerdao.template.yaml` -> template from which we build the yaml
- `package.json - "prepare:yaml" script` -> creates the yaml from the two previous files

I imagine rocketpool won't be deploying its contracts to other networks, so the "only" reason to templatize the yaml would be to keep the consistency with the rest of subgraphs. I would do it :D

Reply:
Also, the folder structure has varied slightly recently. So instead of `/config/networks/ethereum` it will be `/config/deployments/rocket-pool-ethereum/configurations.json`.

You can copy the `prepare:yaml` script from the same badgerdao too, should work as is